Cabal-tests/tests/ParserTests/regressions/public-multilib-1.cabal

```diff
@@ -7,7 +7,7 @@ license:       MIT
 
 library
   default-language: Haskell2010
-  build-depends:    base ^>=4.14, internal
+  build-depends:    base ^>=4.14, internal ^=0
```

Comment:
I think the requirement should be lifted for libraries belonging to the same package